runtime/onert/backend/cpu/ops/ConvolutionLayer.h

```diff
@@ -53,11 +53,13 @@ class ConvolutionLayer : public ::onert::exec::IFunction
   void convQuant8();
 
   void configure(const Tensor *input, const Tensor *kernel, const Tensor *bias,
-                 const ir::PaddingType paddingType, const uint32_t paddingLeft,
-                 const uint32_t paddingRight, const uint32_t paddingTop,
-                 const uint32_t paddingBottom, const uint32_t strideW, const uint32_t strideH,
+                 const ir::Padding padding, const uint32_t paddingLeft, const uint32_t paddingRight,
+                 const uint32_t paddingTop, const uint32_t paddingBottom,
+                 const uint32_t strideWidth, const uint32_t strideHeight,
                  const ir::Activation activation, Tensor *output);
-
+  void configure(const Tensor *input, const Tensor *kernel, const Tensor *bias,
+                 const ir::Padding padding, const ir::Activation activation,
+                 const uint32_t strideWidth, const uint32_t strideHeight, Tensor *output);
```

Comment:
Two `configure`s seems to be fundamentally the same params. Could you unify them to one?